src/pages/preview/[...slug].tsx

```diff
@@ -27,6 +28,9 @@ const ContentPage: NextPage<ContentPageProps> = ({
     relatedContent,
     previewMode,
 }) => {
+    const isStaging = process.env.APP_ENV === 'staging';
+    if (!contentItem || isStaging) return <Error statusCode={404} />;
```

Comment:
I think it might be best to do this in `getServerSideProps` just to ensure that we return a 404 from the server. Here is an example of us doing that in the regular [...slug] page https://github.com/mongodb/devcenter/blob/main/src/pages/%5B...slug%5D.tsx#L106